cmd/nerdctl/fanotify.go

```diff
@@ -0,0 +1,34 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func newFanotifyCommand() *cobra.Command {
+	var execCommand = &cobra.Command{
```

Comment:
`execCommand` -> `fanotifyCommand`

Reply:
`fanotify` seems more like a flag for `run` command than a standalone command. Can we make it part of the run command. E.g., creating something like `run_fanotify_linux.go` and put fanotify related logic here, similar to other parts of run, (run_security_linux.go, run_stargz_linux.go, etc)